programs/clearing_house/src/state/user.rs

```diff
@@ -502,6 +502,10 @@ pub struct UserStats {
     pub last_maker_volume_30d_ts: i64,
     pub last_taker_volume_30d_ts: i64,
     pub last_filler_volume_30d_ts: i64,
+
+    pub bank_0_insurance_lp_shares: u128, // for bank_index = 0
```

Comment:
maybe we make this a vec<> similar to bank_balances

Reply:
only wanted to apply discount based on perp quote insurance, we can opt later on when/if we generalize quote asset?